climada/hazard/tc_tracks.py

```diff
@@ -511,7 +511,6 @@ def read_ibtracs_netcdf(self, provider=None, rescale_windspeeds=True, storm_id=N
             st_penv = xr.apply_ufunc(basin_fun, track_ds.basin, vectorize=True)
 
             # set time_step in hours
-            track_ds.time.values[:1] = track_ds.time[:1].dt.floor('H')
```

Comment:
I was the one who originally added this line long ago, so I can guarantee that it was only a dirty fix for the behavior of equal_timestep. We can safely remove it now.